tasks/main.yml

```
@@ -187,3 +187,26 @@
     - not __realm_join_output.stderr is search("Already joined to this domain")
   changed_when: not __realm_join_output.stderr is
     search("Already joined to this domain")
+
+- name: Configure dynamic DNS updates
+  blockinfile:
+    path: /etc/sssd/sssd.conf
+    state: present
+    block: |
+      dyndns_update = {{ ad_dyndns_update }}
```

```suggestion
      dyndns_update = {{ ad_dyndns_update | string }}
```
does jinja/ansible really use the exact same format for boolean values as sssd.conf?

---

The ini_file module should be used here, some examples can be found in https://github.com/linux-system-roles/tlog/blob/main/tasks/main.yml

---

ini_file would work better. Otherwise, the resulting config might have repeating settings with different values.